app/design/frontend/Diana/default-theme/Magento_Cms/templates/store-selector-sidebar.phtml

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/** @var \Magento\Framework\View\Element\Template $block */
+?>
+<div class="sidebar-selector selector-city" data-bind="scope: 'selectorCity'">
+    <span class="label-store-selector-sidebar">
+        <?= $block->escapeHtml(__('Choose city')) ?>
```

Comment:
@Dianaeva Try not to combine PHP translation and knockout component. Please, use Knockout way for all translations